README.md

```diff
@@ -1,3 +1,125 @@
+<div align="center">
+  <img width="650" alt="Screenshot 2025-01-05 at 4 45 45 PM" src="https://github.com/user-attachments/assets/7a8e38eb-9908-409b-9270-cbb2a97b8be5" />
+</div>
```

Comment:
The image URL in the `src` attribute is a placeholder and may not display correctly. Replace with a valid URL.